packages/blitz/src/index-browser.tsx

```diff
@@ -84,6 +84,5 @@ if (typeof window !== "undefined" && process.env.NODE_ENV === "development") {
 
 export * from "./utils"
 export * from "./types"
-export * from "./utils/enhance-prisma"
```

Comment:
this is a breaking change

Reply:
I introduced this in my latest commit. will revert

Reply:
actually is fine